x/evm/artela/contract/contract_func.go

```diff
@@ -231,7 +243,11 @@ func (k *AspectNativeContract) checkAspectOwner(ctx sdk.Context, aspectId common
 }
 
 func (k *AspectNativeContract) CheckIsAspectOwnerByCode(ctx sdk.Context, aspectId common.Address, code []byte, sender common.Address) (bool, error) {
-	runner, newErr := run.NewRunner(aspectId.String(), code)
+	aspectCtx, ok := ctx.Value(types.AspectContextKey).(*types.AspectRuntimeContext)
+	if !ok {
+		return false, errors.New("CheckIsAspectOwnerByCode: wrap artelatype.AspectRuntimeContext failed")
```

Comment:
Thought this should be called as `unwrap` instead of `wrap`?

Reply:
@dumbeng fixed.